internal/proxy/util.go

```diff
@@ -609,6 +610,74 @@ func validateSchema(coll *schemapb.CollectionSchema) error {
 	return nil
 }
 
+func validateFunction(coll *schemapb.CollectionSchema) error {
+	nameMap := lo.SliceToMap(coll.GetFields(), func(field *schemapb.FieldSchema) (string, *schemapb.FieldSchema) {
+		return field.GetName(), field
+	})
+	// validate function
+	for _, function := range coll.GetFunctions() {
```

Comment:
also check:

- if multiple functions, no duplicate function name
- not allowed for multiple functions to have any same output field
- inputs and outputs of a same function may not overlap

Reply:
add a `checkFunctionParams`.

currently for BM25, check k1 and b in range: k1 [0, 3], b [0, 1]. 

we may shortly add `tokenizer` params check

Reply:
`tokenizer` params check may re-use Jiquan's code